# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/hotfix-kern-cont-yenta.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- linux-6.8/drivers/pcmcia/rsrc_nonstatic.c.vanilla	2024-05-06 18:57:32.163080625 +0200
+++ linux-6.8/drivers/pcmcia/rsrc_nonstatic.c	2024-05-06 19:23:24.746204664 +0200
@@ -364,8 +364,8 @@
 		}
 	}
 
-	dev_dbg(&s->dev, "cs: memory probe 0x%06lx-0x%06lx: %pr %pr %u %u %u",
-		base, base+size-1, res1, res2, ret, info1, info2);
+	/*dev_dbg(&s->dev, "cs: memory probe 0x%06lx-0x%06lx: %pr %pr %u %u %u",
+		base, base+size-1, res1, res2, ret, info1, info2);*/
 
 	free_region(res2);
 	free_region(res1);
